### PR TITLE
Make README.md more interoperable

### PR DESCRIPTION
**Pull request by takenspc**
This PR tweaked README.md as follows:

- Use 4 spaces indentation for code blocks as GitHub seems not to support of tab
- Insert a space after `#` as some markdown implementations require it
- Insert a newline before a list as some markdown implemetations require it

### CODE REVIEW

**Comment by tripu** [comment]
LGTM.

File: README.md [hunk 1]
```diff
@@ -11,10 +11,10 @@ seen at [http://validator.w3.org/checklink](http://validator.w3.org/checklink).
 
 To install the distribution for command line use:
 
-   perl Makefile.PL
```
Comment (astorije) [inline README.md]: Sorry to jump in, but fyi it's actually better to use the 3-backtick+lang+no-space format. Example:

```sh
command $env # comment
```

vs.

    command $env # comment

**Comment by takenspc** [comment]
@tripu, @astorije

Thank you for your comments.
I changed code blocks to make README look nicer on the GitHub.

If it is ok, I'll squash the commits.

**Comment by tripu** [comment]
Thank you, @takenspc. No need to squash commits.

**Pull request merged**